qualang_tools/plot/Fitting/fitting.py

```diff
@@ -350,22 +358,23 @@ def ramsey(x_data, y_data, guess, verbose=False, plot=False, save=False):
         if guess is not None:
             for key in guess.keys():
                 if key == "f":
-                    guess_freq = guess[key] * x_normal
+                    guess_freq = float(guess[key]) * x_normal
```

Comment:
Why do we need to cast into float here ?

Reply:
Since when I tried it I wore something like 
guess = {'a': '5'
               'b':'10}
So in case that someone will write it like this ...

Reply:
Yes but in python it doesn't matter no? `int * float` will always return a `float`

Reply:
But if it is a string, float(guess[key]) will change it to float...

Reply:
Oh right, this is a good point !